### ISSUE TITLE
Local Modifiers on Belts

### ISSUE THREAD

**Issue by 766b** [Kyusung4698/PoE-Overlay#143]
`+493 to Armour` and `+23 to maximum Energy Shield` queried as local modifiers. Accessories (as far as I know) do not have local modifiers.

~~~~~~~~
Rarity: Rare
Foe Lash
Stygian Vise
--------
Requirements:
Level: 68
--------
Sockets: A 
--------
Item Level: 86
--------
Has 1 Abyssal Socket (implicit)
--------
+14 to Strength
+493 to Armour
+23 to maximum Energy Shield
+85 to maximum Life
+10% to Cold and Chaos Resistances (crafted)

**Comment by Kyusung4698** [comment]
Thanks for reporting. This is an edge case i did not handle. Occuring if the local mod is infront of the global mod. It will be fixed with the next version.

**Comment by Kyusung4698** [comment]
Fixed with 0.5.8.
